earth_enterprise/src/fusion/gepublish/geserveradmin.cpp

```diff
@@ -623,6 +631,14 @@ int main(int argc, char* argv[]) {
         throw khException("Target path is not specified.\n");
       }
 
+      if ((enable_poisearch || enable_enhancedsearch) && !publishdb.size()) {
```

Comment:
OK, I ran my tests, and everything looks good, but I'm going to ask for one last thing. Can we move these checks closer to the beginning of main. Right after where the command line arguments are parsed (around line 396) seems like a good place to me. I'd like to have these checks run before (almost) anything else so we can catch error conditions as quickly as possible.